enhancements/operator-hub-check.md

```diff
@@ -0,0 +1,103 @@
+---
+title: operatorhub-bundle-validation
+authors:
+  - "@camilamacedo86"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2021-02-19
+last-updated: 2021-02-19
+status: implementable
+---
+
+# Add checks to OperatorHubValidator
+
+This proposal describes a new checks to be implemented in the [operator-framework/api][oper-api] into the [OperatorHubValidator][operatorhubvalidator].
```

Comment:
Could you rewrite this summary please? I don't understand "...new checks to be implemented in the (api) into the (validator)."? I think maybe it means implement new checks in the (validator) in the (api)" or maybe this can simply not mention API at all?

Reply:
changed. I hope that it addresses your concern.